cadcutils/cadcutils/net/auth.py

```diff
@@ -174,7 +258,7 @@ def _signal_handler(signal, frame):
             # if args.cert_filename is None:
             #    cert_filename = os.path.join(os.getenv("HOME", "/tmp"), ".ssl/cadcproxy.pem")
 
-            cert = get_cert(cert_server=args.cert_server,
+            cert = get_cert(cert_server=args.cert_server, user=args.user,
```

Comment:
I agree we minimize prompting but I think you need to prompt when information is missing.